diesel/src/query_builder/mod.rs

```diff
@@ -31,6 +31,7 @@ mod select_statement;
 mod sql_query;
 mod update_statement;
 pub(crate) mod upsert;
+pub use self::upsert::on_conflict_target_decorations::DecoratableTarget;
```

Comment:
This `pub use` belongs down to the `pub use` block starting in line 37.

Reply:
done